addons/mail/static/src/xml/discuss.xml

```diff
@@ -311,6 +311,9 @@
                     <span class="o_preview_counter">
                         <t t-if="preview.unreadCounter &gt; 0">&amp;nbsp;(<t t-esc="preview.unreadCounter"/>)</t>
                     </span>
+                    <span class="o_preview_expand">
+                        <span class="pl-2 o_discuss_icon o_thread_window_expand fa fa-expand" t-if="preview.documentModel"/>
```

Comment:
You do not want to display the expand icon on mail failures. It should only be displayed on previews related to document threads.

The clean solution would be to extend previews to have a key that tells if they are linked to a document thread, i.e. `isLinkedToDocumentThread`.
That way your condition would simply be `t-if="preview.isLinkedToDocumentThread"`.